csharp/documentation/library-coverage/cwe-sink.csv

```diff
@@ -1,2 +1,2 @@
 CWE,Sink identifier,Label
-CWE-079,html xss,Cross-site scripting
\ No newline at end of file
+CWE-079,html-injection js-injection,Cross-site scripting
```

Comment:
A couple of clarifying questions
(1) Do you know what this file is being used for (where is it consumed)? :-)
(2) Could you remind me what "js" is an abbreviation for in "js-injection"?

Reply:
(1) It is used to create the `CWE-079 Cross-site scripting` column in [coverage.rst](https://github.com/github/codeql/blob/5a1c001f07e55450e458959d960ce196ef5af10f/csharp/documentation/library-coverage/coverage.rst).

(2) "js" is an abbreviation for JavaScript. I'll send you a link in Slack to the conversation that discussed the `xss` to `js-injection` change. Let me know if you think this change doesn't make sense for C# after all.